api/src/main/java/org/apache/iceberg/types/Types.java

```diff
@@ -543,6 +563,122 @@ public int hashCode() {
     }
   }
 
+  public static class GeometryType extends PrimitiveType {
+
+    private final String crs;
+
+    private GeometryType(String crs) {
+      Preconditions.checkNotNull(crs, "CRS cannot be null");
+      this.crs = crs;
+    }
+
+    public static GeometryType get() {
+      return new GeometryType("");
```

Comment:
Please use a valid CRS rather than an empty string. I think that this defaults to CRS84, which seems like a reasonable string to use.

Reply:
We are using `null` for now. Now I believe that using `OGC:CRS84` could be a better choice. See the comments for `GeographyType.toString` below.

Reply:
I think `null` is a good choice for the default.